scrapers/Staxus.yml

```
@@ -0,0 +1,382 @@
+name: "Staxus"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - https://staxus.com/trial/
+    scraper: sceneScraper
+
+performerByURL:
+  - action: scrapeXPath
+    url:
+      - https://staxus.com/trial/
+    scraper: performerScraper
+
+xPathScrapers:
+  sceneScraper:
+    common:
+      $info: //div[@class="text"]
+    scene:
+      Title:
+        selector: //title/text()
+        postProcess:
+          - replace:
+              - regex: 'Staxus Scene - '
+                with: ""
```

Using
```suggestion
      Title: //div[@class="video-descr__title"]//h2
```
as a title saves us the pp